Library/Keyboard.swift

```diff
@@ -35,7 +36,8 @@ public final class Keyboard {
     self.changeObserver.send(value: (
       frame.cgRectValue,
       duration.doubleValue,
-      UIViewAnimationOptions(rawValue: UInt(curve.rawValue))
+      UIViewAnimationOptions(rawValue: UInt(curve.rawValue)),
+      notificationName: notification.name
```

Comment:
Yeah for some reason the height doesn't change, which is why I had to inspect the notification name:

```
Name(_rawValue: UIKeyboardWillShowNotification) frame: (0.0, 409.0, 375.0, 258.0)
Name(_rawValue: UIKeyboardWillHideNotification) frame: (0.0, 667.0, 375.0, 258.0)
```

Also - we should see who this behaves with [custom keyboards](http://stackoverflow.com/a/26004605) 😕